ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/tests/CommandDeduplication.scala

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2020 The DAML Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.api.testtool.tests
+
+import java.util.UUID
+
+import com.daml.ledger.api.testtool.infrastructure.Allocation._
+//import com.daml.ledger.api.testtool.infrastructure.Assertions.assertGrpcError
```

Comment:
Minor: remember to get rid of code that's been commented out. :slightly_smiling_face: